### DISCUSSION TITLE
Custom date display in taskbar

### DISCUSSION THREAD

**Discussion by Caspid** [valinet/ExplorerPatcher#2675] [Ideas]
I like when the date display is "M/d" (do you really need to know what year it is?). I know I can change the  "Short date" format in Windows, but this messes with other things (like when you're in Detailed view in Explorer). It would be amazing if I could specify a custom date display just for the taskbar. Thanks for reading!

![Screenshot 2024-01-10 073230_th](https://github.com/valinet/ExplorerPatcher/assets/2336773/3b10a846-68a7-4cdb-ad05-865e758fd965)

**Comment by pyrates999** [comment]
That is beyond what EP is for.
